src/main/java/com/ecore/roles/web/rest/RolesRestController.java

```diff
@@ -0,0 +1,66 @@
+package com.ecore.roles.web.rest;
+
+import com.ecore.roles.model.Role;
+import com.ecore.roles.service.RolesService;
+import com.ecore.roles.web.RolesApi;
+import com.ecore.roles.web.dto.RoleDto;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import javax.validation.Valid;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+
+import static com.ecore.roles.web.dto.RoleDto.fromModel;
+
+@RequiredArgsConstructor
```

Comment:
**Not a must**
For sure it helps with getting less code, but in my opinion this is a case where it's better have the constructor with the Autowired annotation. It makes the code more readable in spring beans. Once we have both standards in the current code, I suggest to go just with one of them as much as possible.